addons/html_editor/static/src/utils/image.js

```diff
@@ -64,9 +65,8 @@ export function getImageSrc(el) {
         return el.dataset.bgSrc;
     }
     const { getComputedStyle } = el.ownerDocument.defaultView;
-    return getBgImageURLFromURL(
-        backgroundImageCssToParts(getComputedStyle(el)["background-image"]).url
-    );
+    const url = backgroundImageCssToParts(getComputedStyle(el)["background-image"]).url;
+    return url && getBgImageURLFromURL(url);
 }
 
 /**
```

Comment:
General comment (not sure you want to handle them in this PR but so you have the info :) ):
- when you add a background image, its format is "None".
- when changing the quality of a background, there is sometimes a flicker of the "Format".
- We should not be able to change the "Format" of a gif
- When modifying an image, the "Image" `OptionsContainer` is not displayed anymore. I checked and in master, it seems that it destroyed then rebuilt. To be checked but maybe that if we follow the same logic, we can encapsulate the async behavior of `ImageFormatOption` in `onWillStart/onWillUpdateProps` -> that could maybe help to avoid the glitch